lib/include/rmb/motorcontrol/sparkmax/SparkMaxPositionController.h

```diff
@@ -73,9 +74,12 @@ class SparkMaxPositionController : PositionController<DistanceUnit> {
     rev::SparkMaxPIDController::AccelStrategy accelStrategy;
   };
 
-  /** \deprecated
```

Comment:
This shouldn't be deleted. It should be above the constructor and not the struct Follower declaration. Honestly, we need to delete this constructor anyways, so I don't know if we should just go ahead and merge.

Reply:
Git automerge moment™️

Reply:
Take a look now, I think it should be fine.

Reply:
looks good!